fiftyone/utils/openworld_sam.py

```
@@ -0,0 +1,432 @@
+        self.name_or_path = self.parse_string(d, "name_or_path")
+        self.score_threshold = self.parse_number(
+            d, "score_threshold", default=0.5
+        )
+        self.class_names = self.parse_array(d, "class_names", default=None)
+        self.nms_threshold = self.parse_number(d, "nms_threshold", default=0.2)
+        self.top_k = self.parse_int(d, "top_k", default=100)
+        self.validate_config()
+
+    def validate_config(self):
+        if not 0 <= self.score_threshold <= 1:
+            raise ValueError("score_threshold must be in [0, 1]")
```

Ruff (0.15.18)

[warning] Avoid specifying long messages outside the exception class

(TRY003)